Source/MonoGame.Extended/Collections/Poolable.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGame.Extended.Collections
+{
+    public abstract class Poolable : IPoolable
```

Comment:
Well my idea was to extend the `Poolable` with a convience method to return it to the `ObjectPool` instead of doing so directly from the `ObjectPool`.